sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/UnaryPrefixOperatorRepeated.Fixed.cs

```diff
@@ -6,12 +6,13 @@ namespace Tests.Diagnostics
 
     class UnaryPrefixOperatorRepeated
     {
-        static void NonComp(bool b  )
+        static void NonComp(bool bbb)
```

Comment:
is `bbb` used?

Reply:
`bbb` is used a few lines below (the same method is called recursively)